packages/components/menu/src/menu-item.ts

```diff
@@ -199,14 +221,19 @@ export class MenuItem extends ScopedElementsMixin(LitElement) {
     }
 
     this.submenu?.showPopover();
+    this.setAttribute('aria-expanded', 'true');
 
     if (focus) {
       this.submenu?.focus();
     }
   }
 
   #hideSubMenu(): void {
+    if (!this.submenu) {
+      return;
+    }
     this.submenu?.hidePopover();
+    this.setAttribute('aria-expanded', 'false');
```

Comment:
You are setting `aria-expanded` here directly after show/hide popover. An alternative could be to set it from a `beforetoggle` event listener on `this.submenu`??? That way if a user presses Escape, it'll also update the `aria-expanded` attribute?